wpilibj/src/main/java/edu/wpi/first/wpilibj/experimental/command/CommandState.java

```diff
@@ -0,0 +1,39 @@
+package edu.wpi.first.wpilibj.experimental.command;
+
+import edu.wpi.first.wpilibj.Timer;
+
+
+public class CommandState {
```

Comment:
Needs docs.

Does this need to be public if it's for internal state tracking in the scheduler?

Reply:
Nope, this should have been package-private, good catch.